DuckDuckGo/AppLifecycle/AppStates/Foreground.swift

```diff
@@ -443,7 +443,8 @@ struct Foreground: AppState {
             DispatchQueue.main.asyncAfter(deadline: DispatchTime.now() + 0.5) {
                 mainViewController.launchAutofillLogins(openSearch: true, source: source)
             }
-
+        case .openAIChat:
+            AIChatDeepLinkHandler().handleDeepLink(url, on: mainViewController)
```

Comment:
I don't see this as a blocker, but I noticed that this will recreate the AI Chat view when the view is already open, so the history gets wiped out.

Reply:
We have saved chats. This is by design because every time you open duck.ai, it should start with a clean slate (it used to save the state, but a change was requested). That said, the previous conversation is stored in the history.